debian/control

```diff
@@ -25,6 +24,7 @@ Homepage: https://github.com/intel/xpumanager
 Package: libxpum1
 Architecture: amd64
 Depends:
+  libigsc0 (>=0.9.3),
   ${shlibs:Depends},
   ${misc:Depends}
 Recommends:
```

Comment:
I also recommend to move the "recommends" and "suggests" to xpu-smi

Reply:
Added a new commit to move those over